pkg/plugin/pr-sanitizer/event_handler.go

```diff
@@ -43,13 +30,51 @@ var (
 	defaultTypes          = []string{"chore", "docs", "feat", "fix", "refactor", "style", "test"}
 )
 
+// DescriptionShortMessage is message notification for contributor about PR short description content.
+const DescriptionShortMessage = "Hey @%s! It seems that PR description is too short. More elaborated description will be helpful to " +
+	"understand changes proposed in this PR."
+
+// HandleEvent is an entry point for the plugin logic. This method is invoked by the Server when
+// events are dispatched from the /hook service
+func (gh *GitHubLabelsEventsHandler) HandleEvent(log log.Logger, eventType github.EventType, payload []byte) error {
```

Comment:
How about renaming `GitHubLabelsEventsHandler` to `GitHubPRSanitizerEventsHandler` or something similar since we do not do anything with the `Labels` here.